src/modules/fancyzones/FancyZonesLib/trace.cpp

```diff
@@ -212,7 +212,7 @@ void Trace::FancyZones::DataChanged() noexcept
             activeZoneSetInfo += L", custom zone data was deleted";
         }
     }
-
```

Comment:
Why the suppress here? What's the reason. It should have a comment to explain why the code can't be fixed and some better description of the warning that's being surpressed.

Reply:
Good to investigate this one.
I thought it was being generate at the macros level, but in fact the compiler report the wrong line as it is a complex one.
Solved now without supress it.